test/maps/port_space.wmf/scripting/test_portfield_covered_by_tree.lua

```diff
@@ -0,0 +1,9 @@
+run(function()
+   --  This places to tower so that the port just fits the space.
+   map:place_immovable("alder_summer_old", map:get_field(7,10))
+   local ship = create_infrastructure(59, 10)
+   game.desired_speed = 1000
+   wait_for_message("Port Space")
```

Comment:
Please also test with a tree and a young tree placed to two of the main fields of the portspace. Maybe with a rock again at (7,10).